pkg/api/v1/schema.go

```diff
@@ -111,6 +111,11 @@ type DomainSpec struct {
 	Features *Features `json:"features,omitempty"`
 	// Devices allows adding disks, network interfaces, ...
 	Devices Devices `json:"devices"`
+	// Controls whether or not disks will share IOThreads.
+	// Omitting IOThreadsPolicy disables use of IOThreads.
+	// One of: shared, auto
```

Comment:
Can you introduce constants with their own string subtype?

Reply:
My bad, I had misread that the openapi spec explicitly did not support enums, but that was old information. fixed in next push :)